ext/fetch/internal.d.ts

```diff
@@ -1,6 +1,6 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 
-// deno-lint-ignore-file no-explicit-any
+// deno-lint-ignore-file no-explicit-any no-var
```

Comment:
`no-var` should presumably be not applied to `d.ts` files? Or is it one of the things users shouldn't do but we still want to do in our declaration files?

Reply:
It is subjective in `.d.ts` files as it really is only lib files that it is required so things are globally scoped, and most people don't author lib files, but either way, unrelated to this PR.